MathiasWP/e2e-skeleton-project
# Implementing a simple Cypress drag and drop test

- Installed cypress
- Added a package.json scripts entry
- Added a couple of drag and drop tests

How to test:

1. `npm install`
2. Spin up website using `npm run dev`
3. Open cypress using `npm run cypress`
4. Pick a browser and click on `test_drag` entry under integrations. It should run the tests in browser and show status.
